### PR TITLE
max # of moves per turn and territory capture

### PR DESCRIPTION
**Pull request by cakey**
need to clean up tile management and team management...

@Stewart-Taylor

### CODE REVIEW

File: src/client/app.coffee [hunk 1]
```diff
@@ -75,23 +78,59 @@ hexTo3d = ([hexX, hexY]) ->
 uuidToHex = new Map()
 hexToUuid = new Map()
 validHexes = new Set()
+uuidToTile = new Map()
+
+
+# to do: tile manager class
```
Comment (Stewart-Taylor) [inline src/client/app.coffee:84]: What will the Title Manager class be for? Holding all the tiles?

File: src/client/app.coffee [hunk 2]
```diff
@@ -133,10 +172,12 @@ class Player
 
 class GameView
     constructor: ->
+        @movesPerTurn = 4
```
Comment (Stewart-Taylor) [inline src/client/app.coffee:175]: We should probably have a config file or something so we can tweak game mechanics in the future

File: src/client/app.coffee (hunk 1 above)
Comment (cakey) [inline src/client/app.coffee:84]: Yeah, providing an abstraction so we can just deal with the hex coordinates and not the underlying mesh uuids.

Hide away structures like:

```
uuidToHex = new Map()
hexToUuid = new Map()
validHexes = new Set()
uuidToTile = new Map()
```

File: src/client/app.coffee (hunk 2 above)
Comment (cakey) [inline src/client/app.coffee:175]: yeah

File: src/client/app.coffee (hunk 1 above)
Comment (Stewart-Taylor) [inline src/client/app.coffee:84]: That would be much be nicer we can do that in another branch or soemthing

**Comment by Stewart-Taylor** [comment]
Tried it looks really good and is working fine. Plus it looks sick :+1:

**Pull request merged**